Core/GDCore/Extensions/Builtin/Capabilities/AnimatableExtension.cpp

```diff
@@ -44,7 +44,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAnimatableExtension(
          "number",
          "Index",
          _("Animation (by number)"),
```

Comment:
Looking at your first screenshot, and all the "change xxx" actions we have, I still feel that this should be renamed to `Change Animation (by number)` to be consistent

Reply:
This is because I search "change" but there is more action that don't have "change" actually.
I would rather remove "change" everywhere. This way these actions won't get in the way.